### DISCUSSION TITLE
How large is the initial download?

### DISCUSSION THREAD

**Discussion by FunkyPDB** [Chia-Network/chia-blockchain#1421] [Q&A]
At present, I have a download on my first plot of 33k at 448gb. I have been plotting for over 24 hours. My system specs are below. Any help is appreciated.

Ram: 16g
Processor: Intel core i7-7567U CPU @ 3.50GHz
Hard drive is an external drive 2TB HDD Seagate unit

I am currently still on my first plot, phase 2/4 (I think) computing table 5 and about 48 buckets in.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
